codegen/general_test.java

```diff
@@ -0,0 +1,668 @@
+//Do not expect good code it's only a test
+class Main
+{
+    public static void main(String[] args)
+    {
+        A a;
+        B b;
+        D d;
+        boolean tr;
+        int[] i_arr;
+        boolean [] b_arr;
+        int i;
+        int j;
+        int z;
+        a=new A();
+        b=new B();
+        d=new D();
+        {
+            {
+                {
+                    {
+                        {

```

Comment:
Oh I should also mention that things like this should not be tested here because they're not part of codegen, it's part of parsing. I'm not sure if it happens anywhere else in the file.

Thanks again for the request, looking forward to the changes.

Reply:
You are absolutely right, they are not needed and I forgot to remove them after testing different types of code formatting.